workflows/nextflow.config

```diff
@@ -23,6 +23,12 @@ profiles{
       withLabel: bigdisk {
         queue = 'nextflow-batch-bigdisk-queue'
       }
+      withLabel: bigmem {
```

Comment:
While I think this is fine for testing, if we use this for production, we won't be able to adjust depending on the index we are using. If we keep the memory statement in the process (rather than a tag), I think we may find it easier to change on a per-sample basis (different for nuclei vs cells). At this point, I don't think it matters much, and having the label here doesn't hurt anything, so it is fine.